HerPublicWebsite.BusinessLogic/Services/RegularJobs/RegularJobsService.cs

```diff
@@ -45,4 +56,48 @@ public async Task RunNightlyTasksAsync()
             await dataProvider.PersistAllChangesAsync();
         }
     }
+
+    public async Task<IList<ReferralRequest>> GetReferralsPassedTenWorkingDayThresholdWithNoFollowUp()
+    {
+        var startDate = await AddWorkingDaysToDateTime(DateTime.Now, -10);
+        var newReferrals = await dataProvider.GetReferralRequestsWithNoFollowUpAfterDate(startDate);
```

Comment:
⚠️ Aren't we interested in the `referrals` that happened _before_ this threshold, rather than after?

Reply:
Hmm, yes your right. Not sure how I got turned around on this one. Good spot thank you!